ChangeLog.md

```diff
@@ -20,6 +20,8 @@ See docs/process.md for more on how version tagging works.
 
 4.0.13 (in development)
 -----------------------
+- `--modularize` flag was added to `file_packager.py` available when run 
```

Comment:
I wonder if `--modularize` is the right name for this flag?   How about `--export-es6` or `--module` or `--esm`?   I'm not really sure what is best but in emscripten `-sMODULARIZE` does not imply ES6 modules .. so maybe confusing?